kolibri/plugins/example_plugin/assets/example/example_plugin.js

```diff
@@ -1,5 +1,24 @@
 'use strict';
 
-var Kolibri = require('kolibri.core.KolibriCoreFrontEnd');
+var _ = require('lodash');
+var logging = require('loglevel');
 
-module.exports = {};
+var Plugin = require('plugin_base');
+
+var ExamplePlugin = Plugin.extend({
+
+    events: {
+        'something_happened': 'hello_world'
+    },
+    hello_world: function(plugin) {
+        if (plugin === this) {
```

Comment:
Originally, I was triggering this from the 'plugin_registered' event, which also emits the plugin object as the first argument passed to a callback, this was just checking that the plugin in question was registered.

Ultimately, we can parse arbitrary data and arguments into callback functions from events.